packages/location-management/src/ducks/tests/fixtures.ts

```diff
@@ -66,7 +66,7 @@ export const locationUnit2 = {
     type: 'MultiPolygon',
   },
   id: 'a7baf57d-ad31-46d2-8d28-0c81dd306b08',
-  locationTags: [{ id: 2, name: 'Operational Area' }],
+  locationGroups: [{ id: 2, name: 'Operational Area' }],
```

Comment:
i have concerns related to this, should this be locationtags for now or locationgroups? 

Please advise so i can make changes accordingly or leave it as location groups, this rely on the backend payload.

Reply:
@ashfahan

Reply:
as it is a fixture file (usually contain dummy from server) and as far as i know we are receiving locatoinTags from server not locationGroups

Reply:
In that case we don't have any actions in this and all the locations, we will handle that while sending data to backend or receiving data, we change the key for ui

Reply:
yes you dont need to modify these keys